composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
You can't access the injector in the exception handler, that would be a service locator and violate inversion.